MonumentAddons.cs

```diff
@@ -3065,7 +3041,7 @@ private BaseMonument GetClosestMonument(BasePlayer player, Vector3 position)
             if (OnCargoShip(player, position, out cargoShipMonument))
                 return cargoShipMonument;
 
-            return GetClosestMonumentAdapter(position);
+            return _pluginInstance._monumentHelper.GetClosestMonumentAdapter(position);
```

Comment:
Can remove `_pluginInstance.` since this is already inside an instance method of the plugin class.